contracts/hooks/examples/FullRange.sol

```diff
@@ -264,7 +266,7 @@ contract FullRange is BaseHook, ILockCallback {
             if (sender == address(this)) {
                 currency.transfer(address(poolManager), amount);
             } else {
-                IERC20Minimal(Currency.unwrap(currency)).transferFrom(sender, address(poolManager), amount);
+                SafeERC20.safeTransferFrom(IERC20(Currency.unwrap(currency)), sender, address(poolManager), amount);
```

Comment:
hi! please could you use
`using SafeERC20 for IERC20`
and then just token.safeTransferFrom?